### ISSUE TITLE
Separate options to disable editor-mode selecting and dragging individually

### ISSUE THREAD

**Issue by towerofnix** [ScratchAddons/ScratchAddons#3567]
(Separate tracking issue for now-resolved #2380, see https://github.com/ScratchAddons/ScratchAddons/issues/2380#issuecomment-913292558.)

The "Non-draggable sprites in editor" addon is a convenient way to preview how a Scratch project will actually behave when viewed on the player page. Its behavior has two distinct parts:

1. Disable editor-style sprite dragging. (Note sprites which have been specifically made draggable may still be dragged, in the distinct editor-style.)
2. Disable selecting sprites by double clicking.

For completeness and general use, these should be made into separate toggleable options, instead of making all users of the addon use both features.

**Comment by CST1229** [comment]
Maybe split up toggling editor-style dragging to 3 options:
- Editor-style dragging for all sprites
- Project-style dragging for all sprites (no selecting the sprite)
- Project page drag behavior (only drag if sprite is draggable)

**Comment by towerofnix** [comment]
Agreed @CST1229! I forgot about that behavior. The options could be:

- Disable dragging sprites in editor
- Disable selecting sprites after double click
- Disable selecting sprites after drag

**Comment by Samq64** [comment]
Was #6229 good enough?
